docs/pages/model_hub/asr.md

```diff
@@ -49,4 +49,68 @@ Here are some other possible configurations for the Whisper deployment:
             compute_type=WhisperComputeType.FLOAT32,
         ).model_dump(mode="json"),
     )
-    ```
\ No newline at end of file
+    ```
+
+### Diarized ASR
+
+Diarized transcription can be generated by using [WhisperDeployment](./../../reference/deployments.md#aana.deployments.WhisperDeployment) and [PyannoteSpeakerDiarizationDeployment](./../../reference/deployments.md#aana.deployments.PyannoteSpeakerDiarizationDeployment) and combining the timelines using post processing with [ASRPostProcessingForDiarization](./../../reference/processors.md).
+
+Example configuration for the PyannoteSpeakerDiarization model is available at [Speaker Diarization](./speaker_recognition.md) model hub.
```

Comment:
Point to diarization section like `speaker_recognition/#speaker-diarization-sd-models`.

Reply:
Done